1.10.5/rhel7/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
     rpm -ivh http://dev.mysql.com/get/mysql-community-release-el7-5.noarch.rpm
 
 
-RUN yum install -y epel-release cyrus-sasl-devel krb5-libs python-devel mysql-devel make tini build-base git krb5-dev openssl-devel libffi-dev linux-headers nodejs nodejs-npm python3-dev tzdata  wget libaio  zlib-devel libcurl-devel python-setuptools python-virtualenv bzip2-devel openssl-devel ncurses-devel  libxml2-devel libxslt-devel  sqlite sqlite-devel gcc sharutils unzip bash tar gcc-c++ && \
+RUN yum install -y epel-release cyrus-sasl-devel krb5-libs python-devel mysql-devel make tini build-base git krb5-dev openssl-devel libffi-dev linux-headers nodejs nodejs-npm python3-dev tzdata  wget libaio  zlib-devel libcurl-devel python-setuptools python-virtualenv bzip2-devel openssl-devel ncurses-devel  libxml2-devel libxslt-devel  sqlite sqlite-devel gcc sharutils unzip bash tar gcc-c++ nmap-ncat && \
```

Comment:
@ianstanton we could also make these a package-per-line list, like we have in `buster` and `alpine3.10`, just for consistency.

Reply:
https://github.com/astronomer/ap-airflow/blob/master/1.10.5/buster/Dockerfile#L58-L75

Reply:
:+1: I like this idea. Easier to read. Will change!

Reply:
Updated this one